src/main/external-resources/renderers/Google-ChromecastUltra.conf

```diff
@@ -41,12 +41,5 @@ Supported = f:m4a            n:2      a:aac-lc|he-aac|mp3   m:audio/mp4
 Supported = f:wav            n:2      a:lpcm                m:audio/x-wav
 Supported = f:weba           n:2      a:opus|vorbis         m:audio/webm
 
-# Supported image formats:
```

Comment:
@Nadahar I did forget :/
That's said, i added it in the Google-Android-Chromecast renderer because it have a specific resolution limitation; and i didn't remember that your PR handled such case, i added it.

Do #1061 use the `w` and `h` limitations set in the renderer configuration ?